browser.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
 
   loadCountry: function loadCountry () {
     throw new BrowserslistError(
-      'Country statistics is not supported ' +
+      'Country statistics are not supported ' +
```

Comment:
Can you move this to another PR?

Reply:
Sure, done at #502 :-) I've already removed the change from this pull-request, too.